tests/integration/test_metrics.py

```diff
@@ -777,23 +777,10 @@ def test_transaction_metrics_count_per_root_project(
     }
 
 
-@pytest.mark.parametrize(
-    "send_extracted_header,expect_extracted_header,expect_metrics_extraction",
-    [(False, True, True), (True, True, False)],
-    ids=["must extract metrics", "mustn't extract metrics"],
-)
 def test_transaction_metrics_extraction_external_relays(
```

Comment:
Not sure what to do with this test, we already test metrics extraction in the `test_generic_metric_extraction`.